.buildkite/test-template.j2

```diff
@@ -26,7 +26,7 @@ steps:
       - label: "AMD: {{ step.label }}"
         agents:
           queue: amd
-        command: bash .buildkite/run-amd-test.sh "'cd {{ (step.working_dir or default_working_dir) | safe  }} && {{ step.command  or (step.commands | join(' && ')) | safe }}'"
+        command: bash .buildkite/run-amd-test.sh "cd {{ (step.working_dir or default_working_dir) | safe  }} ; {{ step.command  or (step.commands | join(" ; ")) | safe }}"
```

Comment:
This might not fail the bash command inside. If the test failed, the whole command will not exit with 1.

Reply:
I've checked (https://buildkite.com/vllm/ci/builds/6722)
I does fail on the failing test inside.